tests/test_alpaca.py

```diff
@@ -29,24 +28,27 @@
 
 class TestAlpaca(unittest.TestCase):
     def test_to_chatml(self):
-        chatml = pychatml.alpaca.to_chatml(EXPECTED_INPUT, True)
+        converter = AlpacaChatMLConverter(treat_instruction_as_system_prompt=True)
```

Comment:
sorry I really do not like this interface and it feels a lot like Java to me. I think ideally it is super simple to get the right function for a project.

I think json package is a good example to strive for

Reply:
this is probably my fault lol